Gemfile

```diff
@@ -20,34 +24,34 @@ gem 'dalli' # memcached
 gem 'parallel'
 
 gem 'google-api-client'
-gem 'sprockets-derailleur' # Multi-cpu assets precompile
+
+#gem 'sprockets-derailleur', github: 'wjordan/sprockets-derailleur' # Multi-cpu assets precompile
```

Comment:
no, I couldn't manage to get this gem working with Rails 5 in the brief time I looked into it, and since it was only a compile-time performance optimization anyway, I'll take it out for the time being (either until it's updated, or we find/create an alternative setup for parallel Sprockets compilation).